src/applications/gi/tests/components/search/InstitutionSearchForm.unit.spec.jsx

```diff
@@ -75,7 +61,7 @@ describe('<InstitutionSearchForm>', () => {
         search={search}
         autocomplete={autocomplete}
         eligibility={eligibility}
-        filters={filters}
+        filters={() => {}}
```

Comment:
This is replacing an object with a function

Reply:
Added that to make it pass, updated it to just pass the `filters` prop.